.github/pull_request_template.md

```diff
@@ -1,38 +1,58 @@
-> Please remove all the lines starting with '>'
-> Hi there! Thanks for submitting a pull request to Concourse! 
-> If you haven't already, please take a look at our [Code of Conduct](https://github.com/concourse/concourse/blob/master/CODE_OF_CONDUCT.md) and our [Contributing Guide](https://github.com/concourse/concourse/blob/master/CONTRIBUTING.md).
-> 
+> Hi there! Thanks for submitting a pull request to Concourse!
+
+> If you haven't already, please take a look at our [Code of Conduct] and our
+> [Contributing Guide]. There are certain anti-patterns that we try to avoid -
+> check the [Anti-patterns page] to now more.
+
 > To help us review your PR, please fill in the following information.
-> Feel free to delete any sections not applicable to your pull request.
 
-# Existing Issue
-> Is there an existing issue related to this PR? Fill in the issue number as follows: Fixes #1234.
+> Also, before submitting, please remove all the lines starting with '>'.
+
+
+[Code of Conduct]: https://github.com/concourse/concourse/blob/master/CODE_OF_CONDUCT.md
```

Comment:
if someone forgets to remove this, it won't cause any harm - it'll not be rendered anyway.

I find this format easier for someone to parse (compared to inline). Wdyt?

Reply:
neat!